collector/sys_user_summary.go

```diff
@@ -0,0 +1,163 @@
+// Copyright 2022 Eduardo J. Ortega U.
```

Comment:
Same here, this also needs to be `The Prometheus Authors`. Please fix all of these headers.

Reply:
Done